python/xorbits/_mars/dataframe/utils.py

```diff
@@ -682,6 +682,28 @@ def build_df(df_obj, fill_value=1, size=1, ensure_string=False):
     return ret_df
 
 
+# by build a new series class, we can directly use pd.Index(Series) to turn it to Index Object.
+def build_empty_index(dtype, index=None, name=None, gpu: Optional[bool] = False):
+    xpd = cudf if gpu is True else pd
+    series_of_index = build_empty_series(dtype, index, name, gpu)
+    return xpd.Index(series_of_index)
+
+
+def build_index(
```

Comment:
This function is not invoked?

Reply:
I think I just wrote it for symmetric as to the case of build_series